### ISSUE TITLE
Open Government Model

### ISSUE THREAD

**Issue by nullivex** [Flyflap/Flyflap#3]
This fork should also work on solving the issue that caused the problems in the current Flyspray.

1) Open Government through elections and democracy
2) Open control model.

Please see what has just recently happened with Node.js and IO.js which just went through the same power struggle with a corporation trying to take over an Open Source project.

Even if this fork is removed a similar like this should be implemented in Flyspray.

**Comment by peterdd** [comment]
Yes, but I have absolute no experience how to build such a org thing and the time I invest I try to work on the code.

**Comment by scottscreations** [comment]
I don't think it would take much effort. Look at that commit on the ad thing and you can see some comments from long time users. That one guy with 5 years running on flyspray should be on a committee or at least have a voice. I really think an irc meetup a few times a year would do most of the planning and direction of the project. One cool thing Archlinux does is irc bug squashing event. Keep this fork updates though, we can't let flyspray fall.

Everytime i see flyflap now i think flyfap.....haha.

**Comment by peterdd** [comment]
Are you related to archlinux project? They use an flyspray version for their bug tracking and had the wish to ask them some things like their ldap integration for example and how they integrated flyspray with the rest of their website.

I really wish to ship a flyspray1.0beta now and then someone there could take a look for the ldap integration into flyspray.

**Comment by scottscreations** [comment]
I am not, but yeah they use a modified flyspray. I sorta miss that old look ;) something about it.

**Comment by peterdd** [comment]
Yea, that good old 16x16 cute icons ^^
I'm not big fan of this monochrome fonticons that are so modern currently.
But it is currently the simpliest solution for the small dev team of programmers. There is no 
graphic team who creates a complete imagemap with all needed icons and in different sizes for the css media queries (small, normal, retina).

I hope I can add some meaningful (with information) colors in future to flyspray by styling of task properties. 

And compensate the drop of the old blue theme by custom theming of the CleanFS theme.

**Comment by nullivex** [comment]
I liked the old theme too. I helped get the CleanFS implemented which was all done in the older SourceForge repos.

We used to use Flyspray at GigeNET for all our projects. (They might even still use it I am not sure).
